etl/steps/data/garden/un/2024-10-21/census_dates.py

```diff
@@ -0,0 +1,194 @@
+    "(4)": "Population figures compiled from administrative registers in combination with other sources of data, such as sample surveys.",
+    "(5)": "The population by-censuses for 1986 and 1996 were based on one-in-seven sample of the population, while that for 2006 was based on one-in-ten sample of the population.",
+    "(6)": "Enumeration of former Yemen Arab Republic.",
+    "(7)": "Enumeration of former Democratic Yemen.",
+    "(8)": "Through accession of the German Democratic Republic to the Federal Republic of Germany with effect from 3 October 1990, the two German States have united to form one sovereign State. As from the date of unification, the Federal Republic of Germany acts in the United Nations under the designation 'Germany'.",
+    "(9)": "Enumeration of former Federal Republic of Germany.",
+    "(10)": "Combined with agricultural census.",
+    "(11)": "No formal census conducted. A count of numbers of each family group by name, sex, age and whether permanent or expatriate resident is made on 30 or 31 December each year.",
+    "(12)": "A register-based test census was conducted on 5 December 2001 on a sample of 1.2% of the population.",
+    "(13)": "Due to the circumstances, the census was conducted again in 2004.",
+    "(14)": "Census not carried out on the territory of Kosovo and Metohia.",
+    "(15)": "Rolling Census based on continuous sample survey.",
+    "(16)": "Census was planned to be conducted using staggered enumerations province by province. At the end of 2014, only 6 of the 34 provinces had been enumerated.",
+    "(17)": "Traditional decennial census with full field enumeration, and a continuous sample survey.",
+    "(18)": "Population figures compiled from administrative registers and sample surveys while data on housing characteristics are collected through full field enumeration.",
+    "(19)": "Cancelled.",
+}
+
+MONTHS_DICT = {
+    "Jan.": "January",
+    "Feb.": "February",
+    "Mar.": "March",
+    "Apr.": "April",
+    "Jun.": "June",
+    "Jul.": "July",
+    "Aug.": "August",
+    "Sep.": "September",
+    "Oct.": "October",
+    "Nov.": "November",
+    "Dec.": "December",
+}
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load inputs.
+    #
+    # Load meadow dataset.
+    ds_meadow = paths.load_dataset("census_dates")
+
+    # Read table from meadow dataset.
+    tb = ds_meadow["census_dates"].reset_index()
+
+    # remove timeframes from date
+    tb["date"] = tb["date"].apply(lambda x: x.split("-")[1] if "-" in x else x)
+
+    # add comments
+    tb["comment"] = tb.apply(lambda x: get_comment(x["date"], x["country"]), axis=1)
+
+    # clean date and country columns
+    tb["date"] = tb["date"].apply(clean_date)
+    tb["date"] = tb["date"].replace(MONTHS_DICT)
+    tb["country"] = tb["country"].apply(clean_country)
+
+    # convert date to datetime
+    tb["date_as_year"] = tb["date"].apply(date_as_year)
+
+    #
+    # Process data.
+    #
+    tb = geo.harmonize_countries(
```

Comment:
once you remove the empty excluded_countries file you just remove `excluded_countries_file=paths.excluded_countries_path` here